concent_api/concent_api/settings/base.py

```diff
@@ -259,6 +260,16 @@
 # Stored in a 'bytes' array, e.g. b'\xf3\x97\x19\xcdX\xda...'
 # CONCENT_ETHEREUM_PRIVATE_KEY = ''
 
+# A global constant defining the URL of the storage server
+STORAGE_SERVER_INTERNAL_ADDRESS = ''
+
+# A global constant defining Path to a directory where verifier can store files downloaded from the storage server,
+# rendering results and any intermediate files.
+VERIFIER_STORAGE_PATH = ''
+
+# A global constant defining the maximum time (in seconds) rendering a Blender project can take.
+BLENDER_MAX_RENDERING_TIME = 60
```

Comment:
This is pretty low given the default values of our timing constants.

Also, I'm planning another issue (it's still sitting in my backlog) to replace this with a timeout based on protocol times.

Reply:
I think the issue is created, so I am not changing this now.